copy_this/modules/fc/fcoxid2afterbuy/application/controllers/admin/fcafterbuy_orderinfo.php

```diff
@@ -40,4 +40,43 @@ public function fcGetOrderAfterbuyValues() {
 
         return $mReturn;
     }
+
+    /**
+     * Get config parameter for data carrying system
+     *
+     * @return $string
+     */
+    public function getFcAfterbuyLeadSystem()
+    {
+        $oConfig = $this->getConfig();
+        $sValue = $oConfig->getConfigParam('sFcAfterbuyLeadSystem');
+
+        if($sValue == '1') {
```

Comment:
Getting rid of not needed if-statements
`$blReturn = ($sValue == '1');
return $blReturn;`

Reply:
if-statements are kinda self-speaking. Just doing it under protest. ;)